js/src/analysis/PostDataCollector.js

```diff
@@ -318,46 +320,46 @@ PostDataCollector.prototype.saveSnippetData = function( data ) {
 /**
  * Calls the event binders.
  *
- * @param {app} app The app object.
+ * @param {Function} refreshAnalysis Function that triggers a refresh of the analysis.
  *
  * @returns {void}
  */
-PostDataCollector.prototype.bindElementEvents = function( app ) {
-	this.inputElementEventBinder( app );
-	this.changeElementEventBinder( app );
+PostDataCollector.prototype.bindElementEvents = function( refreshAnalysis ) {
```

Comment:
These changes are required in the `TermDataCollector` too.

Reply:
Ouch, yes indeed!